### DISCUSSION TITLE
Issue with Renewing GitHub Education Pack - Stuck on "Please wait as processing can take several seconds."

### DISCUSSION THREAD

**Discussion by Heriniavo-Tiany** [community/community#129612] [GitHub Education]
### Hi everyone,

I'm having trouble renewing my GitHub Education Pack. I have tried submitting various proofs of my academic status, including a dated school ID, a dated receipt, and a dated official transcript, but every time I submit, the process gets stuck on the message "please wait as processing can take several seconds."

I have waited for days, but nothing changes. The only option I have is to cancel and redo the submission, but it still results in the same issue. My submission status remains pending and never progresses.

Has anyone else experienced this issue? If so, how did you resolve it? Are there specific types of documentation that are more likely to be accepted without this delay? Any advice or suggestions would be greatly appreciated.

Thanks in advance for your help!

![image](https://github.com/community/community/assets/92819287/db807a02-9103-406d-be63-a624685b5f30)

**Comment by harzh1** [comment]
Same here with me but it's not evening showing me pending even after uploading documents 2 times
